keps/sig-network/1669-graceful-termination-local-external-traffic-policy/README.md

```diff
@@ -125,8 +125,9 @@ All existing E2E tests for Services should continue to pass.
 
 #### Alpha
 
-* kube-proxy internally tracks the terminating condition of an endpoint.
-* feature is only enabled if the feature gate `EndpointSliceTerminatingCondition` is on.
+* kube-proxy internally tracks the `terminating` and `serving` condition from EndpointSlice
```

Comment:
This change removes all mention of `EndpointSliceTerminatingCondition` but that is the name of the Alpha gate.  Are you proposing to rename the gate?  Why?

Reply:
Yes, I think you suggested we have two separate features gates for this. One for enabling endpointslice conditions and the other for the kube-proxy logic handling terminating endpoints.

Reply:
That makes sense to me, because the terminating condition from EndpointSlice is generally useful outside of kube-proxy as well and I think it would be beneficial for other consumers if we enable it earlier.

Reply:
So to clarify, not renaming the existing feature gate, just proposing a new one. `EndpointSliceTerminatingCondition` is still part of KEP-1672

Reply:
Got it.  That makes more sense.  Thanks.